src/FOHClasses/PhysicalTable.java

```diff
@@ -12,7 +12,8 @@ public class PhysicalTable {
     public PhysicalTable(int tableID, int childSeats) {
         this.tableID = tableID;
         this.childSeats = childSeats;
-        //TODO add this.assignedWaiters and this.tableCollection
+        this.assignedWaiters = new WaiterCollection();
```

Comment:
I believe waiters should be assigned to the BookableTable class instead of the PhysicalTable class. A BookableTable consisting of multiple joined tables may have a single waiter waiting them so it would be unnecessary for each table to have a WaiterCollection when it can be done under BookableTable.